app/ui/design-system/src/lib/Components/SDKCard/index.tsx

```diff
@@ -53,7 +53,12 @@ export function SDKCard({
           </div>
 
           <div className="flex shrink-0 gap-1">
-            <StarIcon className="h-fit text-amber-400" width={14} height={14} />
+            <StarIcon
+              className="text-amber-400"
+              viewBox="0 0 24 24"
```

Comment:
It feels strange to have to explicitly specify the `viewBox` here. If you remove the height/width from the underlying SVG (may need to run `yarn run dev:svg` after), svgr will keep the `viewBox` attribute and the SVG can then  be scaled by specifying height and/or width without needing to also specify `viewBox`. 

I wonder if we should consider configuring svgr to not do this in the first place? It's technically _svgo_ that is removing it -- the `removeViewBox` plugin is enabled by default, and svgr uses svgo under-the-hood.  @10thfloor what do you think? I think all of this changed recently with the repo reconfiguration because I don't think this was happening before.